virt/virt-4-8-release-notes.adoc

```diff
@@ -75,6 +75,7 @@ The SVVP Certification applies to:
 //CNV-12270 CDI can now automatically choose preferred accessMode and volumeMode settings when importing VM disk images.
 
 //CNV-12272 It is now possible to clone virtual machine disks from a Filesystem PVC to a Block PVC and visa versa.
+* xref:../virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume#virt-cloning-vm-disk-into-new-datavolume[Cloning virtual machine disks between different data volume modes] is supported, such as cloning from a persistent volume (PV) with `volumeMode: Block` to a PV with `volumeMode: Filesystem`. However, you can only clone between different volume modes if they are of the `contentType: kubevirt`.
```

Comment:
Suggestion to use active voice and rewrite the note as - "You can now _clone virtual machine disks between different data volume modes_ if they have the content type `kubevirt`. For example, you can clone a persistent volume (PV) with `volumeMode: Block` to a PV with `volumeMode: Filesystem`."